### PR TITLE
Changed id to rackerlabs-blueflood-datasource; Added images for display

### PR DESCRIPTION
**Pull request by ChandraAddala**
1) I changed the id as per the naming convention that is being followed
2) Added images so that we can show some sample screens.
3) I generate the code in this repo from the src code changes of this PR https://github.com/rackerlabs/blueflood-grafana/pull/2

### CODE REVIEW

**Comment by ChandraAddala** [comment]
@goru97 Can you pls take a look?

**Comment by goru97** [comment]
@ChandraAddala Can you please add a description (#What #Why) to this PR and also I can't see the changes you made to datasource.js file (because it's transpiled). Is there a PR against the source code repo (as this is a plugin).

**Comment by ChandraAddala** [comment]
@goru97 Here is the PR for source code changes. https://github.com/rackerlabs/blueflood-grafana/pull/2

File: partials/query.editor.html [hunk 1]
```diff
@@ -1,13 +1,23 @@
...
     <div class="gf-form" ng-show="ctrl.target.textEditor">
-        <input type="text" class="gf-form-input" ng-model="ctrl.target.target" spellcheck="false" ng-blur="ctrl.refresh()"></input>
+        <input type="text" class="gf-form-input" ng-model="ctrl.target.target" spellcheck="false" ng-blur="ctrl.refresh()"/>
     </div>
 
     <div ng-hide="ctrl.target.textEditor">
         <div class="gf-form-inline">
             <div ng-repeat="segment in ctrl.segments" role="menuitem" class="gf-form">
-                <metric-segment segment="segment" get-options="ctrl.getAltSegments($index)" on-change="ctrl.segmentValueChanged(segment, $index)"></metric-segment>
+                <metric-segment property="ctrl.target.target" segment="segment" get-options="ctrl.getAltSegments($index)" on-change="ctrl.segmentValueChanged(segment, $index)"></metric-segment>
```
Comment (goru97) [inline partials/query.editor.html:20]: Why is this change required? Also, this change is not in conjunction with this PR https://github.com/rackerlabs/blueflood-grafana/pull/2

Comment (goru97) [reply partials/query.editor.html:20]: Correction: I see this change in the original src code. maybe the src and this transpiled code  were not in sync when I added them to rackerlabs. Make sure these changes won't blow anything and you still can see the metrics otherwise lgtm.
